### PR TITLE
Bazel 1.0

### PR DESCRIPTION
**Pull request by dslomov**

### CODE REVIEW

**Comment by laurentlb** [comment]
Any `_` should be inside backquotes.

**Comment by dslomov** [comment]
ptal

File: _posts/2019-10-10-bazel-1.0.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+*   [`--incompatible_use_specific_tool_files`](https://github.com/bazelbuild/bazel/issues/8531)
+*   [`--incompatible_windows_bashless_run_command`](https://github.com/bazelbuild/bazel/issues/8240)
+*   [`--incompatible_windows_native_test_wrapper`](https://github.com/bazelbuild/bazel/issues/6622)`
+
+
+## Windows
+
+
+
+*   Genrule now supports `cmd_bash`, `cmd_ps`, `cmd_ba`t attributes. More details at [https://docs.bazel.build/versions/master/be/general.html#genrule.cmd](https://docs.bazel.build/versions/master/be/general.html#genrule.cmd)
```
[the remark this answers is not in the document]
Comment (laurentlb) [reply _posts/2019-10-10-bazel-1.0.md]: Also I don't think the url should be part of the text.

Suggested rephrase:
```
[`genrule`](https://docs.bazel.build/versions/master/be/general.html#genrule) now supports `cmd_bash`, `cmd_ps`, and `cmd_bat` attributes for better integration on Windows.
```

File: _posts/2019-10-10-bazel-1.0.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+*   All rules now have a default `exec_properties` attribute just like the one on a [platform](https://docs.bazel.build/versions/master/be/platform.html#platform.exec_properties) rule. 
+*   All gRPC connections from Bazel will default to TLS enabled.  To disable TLS use the grpc:// scheme in your URIs. Flags affected are: `--remote_cache`, `--remote_executor` and `--bes_backend`. See #8061 for details.
+*   Make either `--subcommands` or `--verbose_failures` imply `--materialize_param_files`
+
+
+## Configurability
+
+
+
+*   `config_setting` can now check multiple values on "--foo=firstVal --foo=secondVal ..."-style flags
```
Comment (laurentlb) [inline _posts/2019-10-10-bazel-1.0.md]: config_setting should link to https://docs.bazel.build/versions/master/be/general.html#config_setting

File: _posts/2019-10-10-bazel-1.0.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dslomov) [reply _posts/2019-10-10-bazel-1.0.md]: Done

File: _posts/2019-10-10-bazel-1.0.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+*   `aapt2` is now enabled by default. To revert to `aapt`, set the `--android_aapt=aapt` flag.
+*   Fixed Windows path issues with `aapt2`. See the [GitHub issue](https://github.com/bazelbuild/bazel/issues/9102) for more information.
+
+
+## C++
+
+
+
+*   `cc_*` rules support non-transitive defines through a `local_defines` [attribute](https://docs.bazel.build/versions/master/be/c-cpp.html#cc_binary.local_defines).
+*   Bazel now supports [ThinLTO](https://clang.llvm.org/docs/ThinLTO.html) builds on Linux for Clang versions >= 6.0. ThinLTO can be enabled through `--features=thin_lto`.
```
[the remark this answers is not in the document]
Comment (dslomov) [reply _posts/2019-10-10-bazel-1.0.md]: Done

File: _posts/2019-10-10-bazel-1.0.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
+---
+layout: posts
+title: "Bazel 1.0"
+authors:
+  - dslomov
+---
+[Bazel 1.0](https://github.com/bazelbuild/bazel/releases/tag/1.0.0) has just been released.
```
[the remark this answers is not in the document]
Comment (dslomov) [reply _posts/2019-10-10-bazel-1.0.md]: Done

File: _posts/2019-10-10-bazel-1.0.md (hunk 2 above)
Comment (dslomov) [reply _posts/2019-10-10-bazel-1.0.md]: Done

File: _posts/2019-10-10-bazel-1.0.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
+---
+layout: posts
+title: "Bazel 1.0"
+authors:
+  - dslomov
+---
+[Bazel 1.0](https://github.com/bazelbuild/bazel/releases/tag/1.0.0) has just been released. 
+[As we announced before](https://blog.bazel.build/2019/06/06/Bazel-Semantic-Versioning.html) 
+we start following semantric versioning for Bazel releases. We'll maintain at least three-month
```
Comment (laurentlb) [inline _posts/2019-10-10-bazel-1.0.md]: semantic

Comment (dslomov) [reply _posts/2019-10-10-bazel-1.0.md]: Done. Thank you!

File: _posts/2019-10-10-bazel-1.0.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+*   [`--incompatible_use_specific_tool_files`](https://github.com/bazelbuild/bazel/issues/8531)
+*   [`--incompatible_windows_bashless_run_command`](https://github.com/bazelbuild/bazel/issues/8240)
+*   [`--incompatible_windows_native_test_wrapper`](https://github.com/bazelbuild/bazel/issues/6622)`
+
+
+## Windows
+
+
+
+*   Genrule now supports `cmd_bash`, `cmd_ps`, `cmd_bat` attributes. More details in [`genrule` documentation](https://docs.bazel.build/versions/master/be/general.html#genrule.cmd)
```
Comment (laurentlb) [inline _posts/2019-10-10-bazel-1.0.md]: My suggestion was not applied?

Comment (laurentlb) [reply _posts/2019-10-10-bazel-1.0.md]: That was my suggested rephrase:

```
* [`genrule`](https://docs.bazel.build/versions/master/be/general.html#genrule) now supports `cmd_bash`, `cmd_ps`, and `cmd_bat` attributes for better integration on Windows.
```

Comment (dslomov) [reply _posts/2019-10-10-bazel-1.0.md]: Ah now I see: I missed the rest of your suggestion beyond the right scroll

**Pull request merged**